torchrl/objectives/ppo.py

```diff
@@ -133,8 +167,17 @@ def __init__(
         normalize_advantage: bool = False,
         gamma: float = None,
         separate_losses: bool = False,
+        advantage_key: str = None,
+        value_target_key: str = None,
+        value_key: str = None,
```

Comment:
value_key depends on the value network. But if I'm not mistaken, we have not set the value network yet. How does this not produce an exception? What if the value_key has a value that matches the value network output key but is not the one usually expected?